includes/Core/Conversion_Tracking/Conversion_Tracking.php

```diff
@@ -245,6 +245,41 @@ public function get_supported_conversion_events() {
 		return array_unique( $events );
 	}
 
+	/**
+	 * Returns enhanced conversion events supported by active providers from the conversion tracking infrastructure.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array Array of supported enhanced conversion events, or empty array.
+	 */
+	public function get_enhanced_conversion_events() {
+		$providers = $this->get_active_providers();
+
+		if ( empty( $providers ) ) {
+			return array();
+		}
+
+		$events = array();
+
+		foreach ( $providers as $provider ) {
+			$provider_class = get_class( $provider );
+			$constant_name  = $provider_class . '::ENHANCED_CONVERSION_EVENTS';
+
+			if ( defined( $constant_name ) ) {
```

Comment:
Let's better define the new public method on those classes that will return those events. This will be much better than using static properties.